install/sessions.csv

```diff
@@ -417,7 +417,8 @@ TEN2,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2016-11-17T2
 TENN,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2017-11-14T22:08:22Z,9999-01-01T00:00:00Z
 TGHO,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2008-12-22T02:30:00Z,2016-08-17T00:22:37Z
 TGHO,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2016-08-17T00:22:38Z,9999-01-01T00:00:00Z
-TGHR,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2006-05-08T21:26:26Z,9999-01-01T00:00:00Z
+TGHR,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2006-05-08T21:26:26Z,2018-05-15T03:41:00Z
```

Comment:
sorry to be a pain, this timestamp shall be `2018-05-15T00:04:31Z` 
same for the entry below